### PR TITLE
Updates based on native Chinese speaker review.

### PR DESCRIPTION
**Pull request by intelkevinputnam**
Signed-off-by: Kevin Putnam <kevin.putnam@intel.com>

### CODE REVIEW

File: source/get-started/cloud-install/qingcloud.rst [hunk 1]
```diff
@@ -25,8 +25,8 @@ This tutorial assumes that you have completed the following configuration steps:
   `website <https://www.qingcloud.com>`_.
 
 
-Select and start |CL| virtual host QingCloud console
-*****************************************************
+Select and start |CL| virtual host with QingCloud console
```
Comment (mvincerx) [inline source/get-started/cloud-install/qingcloud.rst]: Ln 20: `--what is this for?...` I think this is a leftover comment that should be deleted.

File: source/get-started/cloud-install/qingcloud.rst [hunk 2]
```diff
@@ -138,21 +137,21 @@ Apply for a public IP and add it to the virtual
 
       Apply for public IP   
 
-   After clicking the application, the prompt bar will pop up, read it
+   After clicking the apply button, a dialog will pop up, read it
```
Comment (mvincerx) [inline source/get-started/cloud-install/qingcloud.rst]: Add a period after "pop up." Start a new sentence with "Read it.."

File: source/get-started/cloud-install/qingcloud.rst [hunk 3]
```diff
@@ -45,11 +45,11 @@ Select and start |CL| virtual host QingCloud console
 
       Select |CL| to create a virtual host
 
-#. In the configuration selection interface, you can see virtual hosts of
-   different hardware configuration types, such as adjusting the number of CPU
-   cores, memory size, and hard disk and copy backup strategies. Here we will
-   choose the default configuration for the next demonstration. Click the
-   :guilabel:`下一步` (Next) button to go to the network settings interface.
+#. In the configuration selection interface, where you can configure the
```
Comment (mvincerx) [inline source/get-started/cloud-install/qingcloud.rst]: Remove "where" and it's a complete sentence.

File: source/get-started/cloud-install/qingcloud.rst [hunk 4]
```diff
@@ -138,21 +137,21 @@ Apply for a public IP and add it to the virtual
...
-#. On the application for public network IP page, confirm and fill in the
-   relevant information, including the charging mode and bandwidth limit (the 
-   flow rate mode is used in this tutorial and the 2Mbps bandwidth limit is
-   set). After confirming that it is correct, click :guilabel:`提交` (Submit)
-   button.
+#. On the public network IP application page, confirm and fill in the
+   relevant information, including the charging mode and bandwidth upper limit
+   (the charge-by-bandwidth mode is used in this tutorial and the 2Mbps
+   bandwidth limit is set). After confirming that they are correct, click
+   :guilabel:`提交` (Submit) button.
```
Comment (mvincerx) [inline source/get-started/cloud-install/qingcloud.rst]: Ln 171: Simplify run-on sentence: Add a period after "just created." Start a new sentence.

**Comment by intelkevinputnam** [comment]
@mvincerx Thanks for the feedback.

**Comment by mvincerx** [comment]
LGTM. Thanks!

**Pull request merged**